src/components/IstioWizards/IstioWizard.tsx

```diff
@@ -24,14 +37,32 @@ type WorkloadTraffic = {
 type State = {
   showWizard: boolean;
   workloads: WorkloadTraffic[];
+  mtlsMode: string;
+  loadBalancer: string;
 };
 
+const tlsStyle = style({
+  marginLeft: 20,
+  marginRight: 20
+});
+
+const lbStyle = style({
+  marginLeft: 40,
+  marginRight: 20
+});
+
+const loadBalancerSimple: string[] = ['NONE', 'ROUND_ROBIN', 'LEAST_CONN', 'RANDOM', 'PASSTHROUGH'];
+
+const mTLSMode: string[] = ['NONE', 'ISTIO_MUTUAL', 'MUTUAL', 'SIMPLE', 'DISABLE'];
+
 class IstioWizard extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
       showWizard: false,
-      workloads: []
+      workloads: [],
+      mtlsMode: 'NONE',
+      loadBalancer: 'NONE'
```

Comment:
In case you liked the idea of enumerators, here it would go the corresponding constant.